spec/services/bling_spec.rb

```diff
@@ -0,0 +1,444 @@
+                'volumes' => [
+                  {
+                    'volume' => {
+                      'id' => '3',
+                      'idServico' => '34',
+                      'idOrigem' => '345',
+                      'servico' => 'SEDEX CONTRATO AG',
+                      'codigoServico' => '03456',
+                      'codigoRastreamento' => 'AB12345678',
+                      'valorFretePrevisto' => '13.43',
+                      'remessa' => {
+                        'numero' => '12321313',
+                        'dataCriacao' => '2020-12-21'
+                      },
+                      'dataSaida' => '2020-12-20',
+                      'prazoEntregaPrevisto' => '1',
+                      'valorDeclarado' => '0.00',
+                      'avisoRecebimento' => false,
+                      'maoPropria' => false,
+                      'dimensoes' => {
+                        'peso' => '1.700',
+                        'altura' => '0',
+                        'largura' => '0',
+                        'comprimento' => '0',
+                        'diametro' => '0'
+                      },
+                      'urlRastreamento' => 'www2.correios.com.br'
+                    }
+                  }
+                ],
+                'servico_correios' => 'SEDEX CONTRATO AG'
+              },
+              'itens' => [
+                {
+                  'item' => {
+                    'codigo' => '1',
+                    'descricao' => 'PRODUTO',
+                    'quantidade' => '2.0000',
+                    'valorunidade' => '41.0000000000',
+                    'precocusto' => '22.4400000000',
+                    'descontoItem' => '0.00',
+                    'un' => 'UN',
+                    'pesoBruto' => '1.02000',
+                    'largura' => '168',
+                    'altura' => '241',
+                    'profundidade' => '89',
+                    'descricaoDetalhada' => '',
+                    'unidadeMedida' => 'm',
+                    'gtin' => '12314324343'
+                  }
+                }
+              ],
+              'parcelas' => [
+                {
+                  'parcela' => {
+                    'idLancamento' => '0',
+                    'valor' => '96.60',
+                    'dataVencimento' => '2020-12-26 00:00:00',
+                    'obs' => '',
+                    'destino' => 1,
+                    'forma_pagamento' => {
+                      'id' => '0',
+                      'descricao' => 'Dinheiro',
+                      'codigoFiscal' => '1'
+                    }
+                  }
+                }
+              ],
+              'codigosRastreamento' => {
+                'codigoRastreamento' => 'AB12345678'
+              }
+            }
+          }
+        ]
+      }
+    }
+  end
+  let(:status) { 'Atendido' }
+
+  before do
+    ENV['HUB_SCHEME'] = 'https'
+    ENV['HUB_HOST'] = 'hub.vnda.com.br'
+
```

Comment:
Já que usou para um, usa para os outros.

```suggestion
    stub_const('Vnda::Hub::HUB_SCHEME', 'https')
    stub_const('Vnda::Hub::HUB_HOST', 'hub.vnda.com.br')

```

Reply:
Alteração feita